.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,5 +15,29 @@ is merged. See https://github.com/blog/1506-closing-issues-via-pull-requests
 #### What does this implement/fix? Explain your changes.
 
 
+#### Checklist
+
+- [ ] Followed the [coding-guidelines](#guidelines).
+- [ ] Give your pull request a helpful title that summarizes what your contribution does.
+- [ ] Link your pull request to the issue (see: [closing keywords](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) for an easy way of linking your issue)
+- [ ] All public methods should have informative docstrings with sample usage presented as doctests when appropriate.
+- [ ] At least one paragraph of narrative documentation with links to references in the literature (with PDF links when possible) and the example.
+- [ ] If your feature is complex enough that a doctest is insufficient to fully showcase the utility, consider creating a Jupyter notebook to illustrate use instead.
+- [ ] All functions and classes must have unit tests. These should include, at the very least, type checking and ensuring correct computation/outputs.
+- [ ] All functions and classes should be rigorously typed with Python 3.5+ 
+  [`typehinting`](https://docs.python.org/3/library/typing.html). Validate your typehinting by running `mypy ./graspy`
+- [ ] All code should be automatically formatted by `black`. You can run this formatter by calling:
+  ```bash
+  pip install black
+  black path/to/your_module.py
+  ```
+- [ ] Ensure all tests are passing locally using `pytest`. Install the necessary
+  packages by: 
+
+  ```bash
+  pip install pytest pytest-cov
+  pytest
```

Comment:
@bdpedigo
unrelated to this issue, but this breaks for me in different ways, depending on where i run it from.

if i run it from tests, i get #507 
if i run it from root of the repo, i get:
```
====================================================== ERRORS =======================================================
___________________________________________ ERROR collecting docs/conf.py ___________________________________________
docs/conf.py:93: in <module>
    import sphinx_rtd_theme
E   ModuleNotFoundError: No module named 'sphinx_rtd_theme'
============================================== short test summary info ==============================================
ERROR docs/conf.py - ModuleNotFoundError: No module named 'sphinx_rtd_theme'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================= 1 error in 3.84s ==================================================
```

Reply:
i see that we do recommend to install for devs it in readme

Reply:
installed and still break.

Reply:
@alyakin314 - what's the break error you're getting?  I had issues running `make doc` on macOS as well.